packages/web3/tsconfig.json

```diff
@@ -4,5 +4,5 @@
         "baseUrl": "./src" /* Specify the root folder within your source files. */,
         "outDir": "./dist" /* Specify an output folder for all emitted files. */
     },
-    "include": ["src/**/*"]
+    "include": ["src/**/*", "tests/**/*"]
```

Comment:
Can move this into a separate PR if preferred. Otherwise it may make sense to keep this here because the context is, Promise.any and AggregateError were undefined and that's why we made this change.